java/main/src/main/java/com/saucelabs/saucebindings/options/SauceLabsOptions.java

```diff
@@ -148,20 +142,12 @@ public String getBuild() {
     return build != null ? build : CITools.getBuildName() + ": " + CITools.getBuildNumber();
   }
 
-  /**
-   * @deprecated This method is no longer supported
-   * @return whether CI is accounted for in code
-   */
-  @Deprecated
-  public boolean isKnownCI() {
-    return CITools.getBuildName() != null;
+  public Map<String, Object> getCustomData() {
+    customData.putIfAbsent("sauce-bindings", "java"); // might be set by plugin already
```

Comment:
This needs to set the bindings version as well.

Reply:
Good call... So maybe like this?
```
sauce-bindings: "java"
sauce-bindings-java:  "2.0.1"
sauce-bindings-junit5: "2.0.0"
```

Reply:
https://github.com/saucelabs/sauce_bindings/pull/334/commits/c8b3565b0133c72108f297d14061ac9238c64168 & https://github.com/saucelabs/sauce_bindings/pull/334/commits/648c7f1eee94a7d3dd44fb79ac41d0018d9cbc43